ci/run-k8s-e2e-tests.sh

```diff
@@ -28,13 +28,16 @@ function echoerr {
 RUN_CONFORMANCE=false
 RUN_WHOLE_CONFORMANCE=false
 RUN_NETWORK_POLICY=false
+RUN_SIGNETWORK=false
 RUN_E2E_FOCUS=""
 RUN_E2E_SKIP=""
 KUBECONFIG_OPTION=""
 DEFAULT_E2E_CONFORMANCE_FOCUS="\[Conformance\]"
 DEFAULT_E2E_CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]"
 DEFAULT_E2E_NETWORKPOLICY_FOCUS="\[Feature:NetworkPolicy\]"
 DEFAULT_E2E_NETWORKPOLICY_SKIP=""
+DEFAULT_E2E_SIGNETWORK_FOCUS="\[sig-network\]"
+DEFAULT_E2E_SIGNETWORK_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[GCE\]|\[Ingress\]|\[NetworkPolicy\]|\[IPv6\]|\[Feature:.+\]|\[Feature:IPv6DualStack\]|\[Feature:IPv6DualStackAlphaFeature\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]|should create pod that uses dns|should provide Internet connection for containers"
```

Comment:
Why it needs to list other sigs in skip when it already focus on sig-network?

Reply:
Thanks, I have removed irrelevant items in skip.